assets/js/components/notifications/UserInputSettings.js

```diff
@@ -67,7 +67,7 @@ export default function UserInputSettings( {
 		select( MODULES_ANALYTICS ).isGatheringData()
 	);
 
-	if ( isUserInputCompleted ) {
+	if ( isUserInputCompleted || isUserInputCompleted === undefined ) {
```

Comment:
Just a cosmetic preference. It's better to check for not resolved and then the resolved value. I have pushed the changes 😃